README.md

```diff
@@ -51,6 +53,55 @@ func main() {
 }
 ```
 
+See the [`pgxn_meta` docs on docs.rs] for complete details.
+
+Installation
+------------
+
+There are several ways to install `pgxn_meta`.
+
+### Use `ubi`
+
+Install the [universal binary installer (ubi)](https://github.com/houseabsolute/ubi) and
+use it to install `pgxn_meta` and many other tools.
+
+``` console
+$ ubi --project pgxn/meta --in ~/bin
```

Comment:
Curious when/why we would use `ubi` instead of just `cargo`? I'm asking out of ignorance because I have never used ubi.

Reply:
Oh I meant to include Cargo instructions too. `ubi` installs binaries from GitHub, so it's fast.

Reply:
Oh, the Cargo instructions are below. Phew!